Kickstarter-iOS/Views/Controllers/SignupViewController.swift

```diff
@@ -95,24 +72,48 @@ internal final class SignupViewController: UIViewController, MFMailComposeViewCo
       |> signupButtonStyle
   }
 
-  internal override func bindViewModel() {
-    self.emailTextField.rac.becomeFirstResponder = self.viewModel.outputs.emailTextFieldBecomeFirstResponder
-    self.newsletterSwitch.rac.on = self.viewModel.outputs.setWeeklyNewsletterState
-    self.passwordTextField.rac.becomeFirstResponder =
-      self.viewModel.outputs.passwordTextFieldBecomeFirstResponder
-    self.signupButton.rac.enabled = self.viewModel.outputs.isSignupButtonEnabled
+  fileprivate let environmentLoggedInProperty = MutableProperty(())
```

Comment:
Do you still tend to prefer `MutableProperty<T>` over using the `observer` from `Signal<T, E>.pipe()`? I know there was a memory leak issue at some stage but I think that's since been fixed? (don't have a link handy).

Reply:
It's just been awhile 😅. Been in Rx-land for current projects. I imagine either should work just fine, so if `pipe` saves even more boilerplate and avoids unnecessary optionality I'd definitely go for it!